config.toml

```diff
@@ -43,6 +43,8 @@ enable = true
 [delegate]
 enable = true
 enable_deploy_identity = true
+## The list of accepted fees for delegating a meta-tx
+## example for accepting 0 fees: [{'base_fee'=0, 'gas_price'=0}] other keys are 'currency_network' and 'fee_recipient'
```

Comment:
It would be nice if we could have a test, that tests that our suggested example config is also working. 
I did that by removing single # in the example file, maybe you can adjust it, that it can also work int his case.